infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -297,6 +297,7 @@ public void reset() {
     @Override
     public void reboot() {
```

Comment:
Oh oops, should I just change the method to synchronized then? (even though reboot isn't actually called anywhere).